apidoc/Titanium/UI/Label.yml

```diff
@@ -166,8 +166,13 @@ properties:
     summary: Minimum font size when the font is sized based on the contents. 
     description: |
         Enables font scaling to fit and forces the label content to be limited to a single line.
+        
+        On Android, setting this property will force the `ellipsize` property to be
```

Comment:
Yes, we can link it. But the default value suggestion should not be done, since the default is undefined, so behaving as not set. Otherwise it confuse the developers if they should set `TEXT_ELLIPSIZE_*` constants instead of font-sizes.